modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractGoCodegen.java

```diff
@@ -128,9 +128,15 @@ public AbstractGoCodegen() {
         // map[string]interface{}, whereas an arbitrary type is implemented
         // in golang as interface{}.
         // See issue #5387 for more details.
-        typeMapping.put("object", "map[string]interface{}");
-        typeMapping.put("interface{}", "interface{}");
-        typeMapping.put("AnyType", "interface{}");
+        typeMapping.put("object", "ObjectType");
```

Comment:
There were some discussions before on what `object` should map to in Go:
- https://github.com/OpenAPITools/openapi-generator/pull/3878
- https://github.com/OpenAPITools/openapi-generator/issues/3277#issuecomment-517588291